frontend/src/__tests__/cypress/cypress/pages/components/TableToolbar.ts

```diff
@@ -0,0 +1,19 @@
+class TableToolbar {
```

Comment:
This should probably extend Contextual so that we can target a specific table on the page.